crates/bitwarden-vault/src/collection_client.rs

```diff
@@ -56,6 +62,13 @@ pub struct CollectionViewNodeItem {
     node_item: NodeItem<CollectionView>,
 }
 
+#[cfg_attr(feature = "wasm", derive(Tsify, Serialize, Deserialize))]
+#[cfg_attr(feature = "wasm", tsify(into_wasm_abi, from_wasm_abi))]
```

Comment:
We usually combine ´cfg_attr` with the same condition into a single.

```suggestion
#[cfg_attr(
    feature = "wasm",
    derive(Tsify, Serialize, Deserialize),
    tsify(into_wasm_abi, from_wasm_abi)
)]
```